profiles/common/themes/ec_resp/template.php

```diff
@@ -477,7 +477,7 @@ function ec_resp_preprocess_html(&$variables) {
     if (theme_get_setting('enable_interinstitutional_theme')) {
       $variables['head_title'] = t('EUROPA - !title', array('!title' => $title));
     }
-    else {
+    elseif (arg(0) !== 'splash') {
```

Comment:
Hi,

I've reverted this part. If the theme wants to force the title we should allow it.